consensus/snapshot/snapshot.h

```diff
@@ -37,26 +30,50 @@ typedef struct state_entry_t {
 
 typedef state_entry_t *state_map_t;
 
+typedef struct snapshot_conf_s {
+  uint64_t timestamp;
+  uint64_t signature_index;
+  uint64_t signature_depth;
+  tryte_t signature_pubkey[HASH_LENGTH_TRYTE];
```

Comment:
shouldn't we use flex_trits here?

Reply:
It initially was, let me remember why I changed 😅

Reply:
done